vardict.pl

```diff
@@ -2070,7 +2079,7 @@ sub toVars {
 		}
 	    }
 	}
-	@var = sort { $b->{ qual } * $b->{cov} <=> $a->{ qual } * $a->{cov}; } @var;
+	@var = sort { $b->{ qual } * $b->{cov} <=> $a->{ qual } * $a->{cov} || $a->{ n } cmp $b->{ n }; } @var;
```

Comment:
Additional sort by description strings. When we have no order it causes in the different results of $genotype2 field between runs.